src/GraphQL.sln

```diff
@@ -64,6 +64,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Docs", "Docs", "{EB92DFCF-A
 		..\docs2\site\docs\getting-started\variables.md = ..\docs2\site\docs\getting-started\variables.md
 	EndProjectSection
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "GraphQL.Serialization.Newtonsoft", "GraphQL.Serialization.Newtonsoft\GraphQL.Serialization.Newtonsoft.csproj", "{A8905FD0-EE1C-4847-AFA6-B9EB590FEE79}"
```

Comment:
In the future this project should be moved to a separate repository.

Reply:
I think it should just stay in this one.  I've been thinking about also moving the server project into the main repo so there isn't as much churn between the two.

Reply:
The advantages of this approach are simplicity. The drawbacks are in spurious changes, when new versions of packages will be released in which nothing was actually changed. In fact, this is already happening with Playground/Graphiql/Voyager in the server project now.

Reply:
Understood.  I think there has currently been not enough "compatibility" between this core project and the server project.  Things that don't need to change together, such as the Playground etc. make sense to be independent.  If we can get proper abstractions for the serialization, I could see those being in different repos.

Reply:
Of course, let it stay in this project for now. I think that by the time of version 3 release we can also reorganize the structure of the repositories, not earlier.